etc/notes/CHANGES_5.0.0.md

```diff
@@ -0,0 +1,84 @@
+# Changes in v5
```

Comment:
question: do we want a release guide here?  We could just link to the driver's release notes, since the legacy driver v5 will inherit all breaking changes from driver v5.

Reply:
I would link to the driver in this repo's main readme file. We've made our intention of this package clear, supporting callbacks, so we can say all other changes follow from the driver

Reply:
Done.

Reply:
I don't think we need a separate file that just links back to the driver. Maybe just a section / mention in the [main readme](https://github.com/mongodb-js/nodejs-mongodb-legacy/blob/main/readme.md)

In the same vein as what we have on the driver:
```md
**Upgrading to version 5? Take a look at our [upgrade guide here](https://github.com/mongodb/node-mongodb-native/blob/HEAD/etc/notes/CHANGES_5.0.0.md)!**
```

Reply:
So for every major release would we add a new bullet to the list? 

```
**Upgrading to version 7? Take a look at our [upgrade guide here](https://github.com/mongodb/node-mongodb-native/blob/HEAD/etc/notes/CHANGES_7.0.0.md)!**
**Upgrading to version 6? Take a look at our [upgrade guide here](https://github.com/mongodb/node-mongodb-native/blob/HEAD/etc/notes/CHANGES_6.0.0.md)!**
**Upgrading to version 5? Take a look at our [upgrade guide here](https://github.com/mongodb/node-mongodb-native/blob/HEAD/etc/notes/CHANGES_5.0.0.md)!**
```

Reply:
When we released 4x we changed it to ask about upgrading from 3x, before it said 2x, so I think we can just have a bullet for the prev major

Reply:
https://github.com/mongodb/node-mongodb-native/blob/v3.7.3/README.md?plain=1#L7

Reply:
Okay, I think that makes sense.  I'll remove the upgrade guide I added and link to it in the readme

Reply:
Didn't say 2x I guess but its just pointing to what you need to know about the current major

Reply:
If we're in agreement we still want to move this note to the main readme and remove this file